lab/src/Microsoft.ServiceModel.Syndication/src/XmlWriterWrapper.cs

```diff
@@ -0,0 +1,337 @@
+﻿using System;
+using System.Threading.Tasks;
+using System.Xml;
+
+namespace Microsoft.ServiceModel.Syndication
+{
+    internal class XmlWriterWrapper : XmlWriter
+    {
```

Comment:
Similar changes to what needs to be done to XmlReaderWrapper need to be done here.